tasks/test-project/tasks.js

```diff
@@ -290,10 +290,13 @@ async function apiTasks(outputPath, { verbose, linkWithLatestFwBuild }) {
     // Update src/lib/auth to return roles, so tsc doesn't complain
     const libAuthPath = `${OUTPUT_PATH}/api/src/lib/auth.ts`
     const libAuthContent = fs.readFileSync(libAuthPath, 'utf-8')
-    const newLibAuthContent = libAuthContent.replace(
-      'select: { id: true }',
-      'select: { id: true, roles: true }'
-    )
+
+    const newLibAuthContent = libAuthContent
+      .replace('select: { id: true }', 'select: { id: true, roles: true }')
+      .replace(
+        'const currentUserRoles = context.currentUser?.roles',
+        'const currentUserRoles = context.currentUser?.roles as string | string[]'
```

Comment:
This is to prevent typescript complaining on the smoke tests. TS knows from Primsa types that currentUser.roles can never be a string[] in the test project, so part of the code gets marked as an error.